test/membrane_file/sink_multi_test.exs

```diff
@@ -25,8 +21,6 @@ defmodule Membrane.File.Sink.MultiTest do
 
   setup :state_and_ctx
 
-  setup :verify_on_exit!
```

Comment:
Why have you deleted this?

Reply:
`use Membrane.File.TestCaseTemplate` already sets this up